pom.xml

```diff
@@ -68,53 +78,48 @@
     </profiles>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <version>${maven.compiler.version}</version>
+                    <configuration>
+                        <source>${java.version}</source>
+                        <target>${java.version}</target>
+                        <forceJavacCompilerUse>true</forceJavacCompilerUse>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.17</version>
+                <version>${plugin.surefire.version}</version>
                 <configuration>
-                    <argLine>-Xmx2G -Xms2G</argLine>
-                    <skipTests>${skipTests}</skipTests>
+                    <argLine>-Xms1024m -Xmx1024m</argLine>
+                    <includes>
+                        <include>${test.includes}</include>
+                    </includes>
                     <excludes>
                         <exclude>${test.excludes}</exclude>
                     </excludes>
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>cobertura-maven-plugin</artifactId>
-                <version>2.7</version>
-                <configuration>
-                    <check>
-                        <packageRate>68</packageRate>
-                        <fileRate>55</fileRate>
-                        <classRate>45</classRate>
-                        <methodRate>29</methodRate>
-                        <lineRate>22</lineRate>
-                        <conditionalRate>15</conditionalRate>
-                        <haltOnFailure>false</haltOnFailure>
-                    </check>
-                    <instrumentation>
-                        <excludes>
-                            <exclude>org/kuali/**/*.class</exclude>
-                            <exclude>com/rsmart/**/*.class</exclude>
-                        </excludes>
-                    </instrumentation>
-                </configuration>
-            </plugin>
-            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
why is this needed with java 8 version?

Reply:
The compiler plugin's addition of "groupId" may have been added so that the related configuration in the "pluginManagement" section above could properly map to the plugin setup here. I would suggest keeping this change in place, so that the POM will properly refer to the "maven-compiler-plugin" using a group ID and artifact ID pair.